indico/modules/rb/tasks.py

```diff
@@ -111,9 +62,17 @@ def roombooking_occurrences():
 
     try:
         for occ in occurrences:
-            notify_upcoming_occurrence(occ)
-            occ.notification_sent = True
-            if occ.reservation.repeat_frequency == RepeatFrequency.DAY:
-                occ.reservation.occurrences.update({'notification_sent': True})
+            same_user_occurrences = []
+            for other in occurrences:
```

Comment:
And please pay attention to the number of queries that are generated by this task, it may need some joined/subquery loads.